test/woodbury_pd_mat.jl

```diff
@@ -39,6 +39,12 @@
         end
     end
 
+    @testset "*" begin
+        c = 2.0
+        @test c * W == W * c
+        @test c * W_dense ≈ c * W atol=1e-6
```

Comment:
Does the tolerance need to be this high?

Reply:
do you mean the atol should be bigger or smaller? It should be really just numerical noise.

Reply:
Smaller, have you tried `1e-9`?

Reply:
yep, I tried. It passed, although I'm not very sure about whether this is very robust for other random numbers